_includes/configuration/table_clone_elements.md

```diff
@@ -0,0 +1,16 @@
+### `table_clone_elements`
+
+This option enables you to specify which elements should be cloned as empty children when inserting rows/columns to a table. By default it will clone these '`strong` `em` `b` `i` `span` `font` `h1` `h2` `h3` `h4` `h5` `h6` `p` `div`' into new cells.
```

Comment:
Table plugin is also using pronouns in some cases it seems.

Reply:
yeah, pls ignore for this PR :disappointed: